.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+dist: xenial
+language: generic
+env:
+  - PDK=release PDK_FRONTEND=noninteractive
+before_install:
+  - "curl -k https://raw.githubusercontent.com/puppetlabs/pdk-templates/master/.travis/install_pdk.sh | bash"
+script:
+  - "pdk convert -f; pdk validate yaml"
+branches:
+  only:
+    - master
```

Comment:
Seems like this should be "production", yes? Given how branches on this repo are configured